src/main/webapp/order.html

```diff
@@ -28,7 +28,7 @@
 
     <div> Enter Beverage: </div>
     <form>
-    <textarea id= "beverageRequested"> </textarea>
+    <textarea id= "beverageRequested"></textarea>
```

Comment:
Given the time constraints, could we manually test this functionality, and attach some screenshots to this PR?